crates/bevy_asset/src/io/file_asset_io.rs

```diff
@@ -53,6 +53,8 @@ impl FileAssetIo {
     pub fn get_root_path() -> PathBuf {
         if let Ok(manifest_dir) = env::var("CARGO_MANIFEST_DIR") {
             PathBuf::from(manifest_dir)
+        } else if let Ok(env_bevy_asset_root) = env::var("BEVY_ASSET_ROOT") {
```

Comment:
I wonder if `BEVY_ASSET_ROOT` should be checked first, since `CARGO_MANIFEST_DIR` will always be set (AFAIK) when `cargo run` is used

Reply:
I actually agree, I thought I saw a comment mentioning the order I implemented, but it was actually not referring to this env variable, proposed implementation existed later.

https://github.com/bevyengine/bevy/pull/1801#issuecomment-819020645